packages/snowpack/src/util.ts

```diff
@@ -10,7 +10,7 @@ import mkdirp from 'mkdirp';
 import open from 'open';
 import path from 'path';
 import rimraf from 'rimraf';
-import {SnowpackConfig} from './config';
```

Comment:
This removes `util`’s dependency on `config`! 🎉  IMO this is the main advantage.